3-binary-data/package.json

```diff
@@ -38,9 +38,9 @@
     ]
   },
   "dependencies": {
+    "@google-cloud/storage": "^0.1.1",
```

Comment:
The latest version of `@google-cloud/storage` is `0.2.0`. Are you wanting to upgrade in this PR?